tests/smb/smb_operations.py

```diff
@@ -937,3 +937,22 @@ def remove_sub_vol_group(installer, cephfs_vol, smb_subvol_group, smb_subvols):
         raise CephadmOpsExecutionError(
             f"Fail to remove sub volume and sub volume group, Error {e}"
         )
+
+
+def config_smb_images(installer, samba_image, samba_metrics_image):
+    """Configure smb images
+    Args:
+        installer (obj): Installer node obj
+        samba_image (str): Samba server image
+        samba_metrics_image (str): Samba metrics image
+    """
+    try:
+        # Configure samba server image
+        cmd = f"cephadm shell -- ceph config set mgr mgr/cephadm/container_image_samba {samba_image}"
```

Comment:
@MohitBis, can we use this method [1] instead?

[1] https://github.com/red-hat-storage/cephci/blob/main/tests/misc_env/haproxy.py#L116